compiler/rustc_passes/messages.ftl

```diff
@@ -786,6 +786,7 @@ passes_used_compiler_linker =
 
 passes_used_static =
     attribute must be applied to a `static` variable
+    .label = the kind is `{$target}`
```

Comment:
Suggestion: avoid the backticks because the kind is not (necessarily) a keyword

```suggestion
    .label = but this is a {$target}
```

Reply:
Done. Thank you.